nixos/modules/services/misc/passopolis.nix

```diff
@@ -0,0 +1,63 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.passopolis;
+in
+{
+
+  ###### interface
+
+  options = {
+
+    services.passopolis = {
+
+      enable = mkOption {
+        default = false;
+        description = "
+          Enable passopolis password manager service.
+        ";
+      };
+
+      db_url = mkOption {
+        default = "jdbc:postgresql://localhost:5432/passopolis";
```

Comment:
From what I can tell, the app doesn't support authenticated database url's at this time. I'm not a java programmer, so I could be wrong. Where would you persist the credentials if it could potentially contain user/password?